.vscode/settings.json

```diff
@@ -3,7 +3,6 @@
         "*.mdx": "markdown"
     },
     "files.exclude": {
-        "**/.docusaurus": true,
-        "build": true
```

Comment:
Why are we no longer excluding the build folder from VScode settings?

Reply:
I can revert if preferred. I find it awkward to have to browse the build output through another means (system file browser, terminal). Most people won't be running `build` so shouldn't see the output in most cases?

Reply:
Right, that makes sense. The part I find annoying is that build content then gets included in searches, so could we add the build folder to the search exclude to prevent that?

```json
"search.exclude": {
 "build": true
}
```